pyeapi/eapilib.py

```diff
@@ -309,8 +309,7 @@ def authentication(self, username, password):
             _auth_bin = base64.encodebytes(_auth_text.encode())
             _auth = _auth_bin.decode()
         else:
-            # For Python 2.7
-            _auth = str(base64.encodestring(_auth_text))
+            raise Exception("Incompatible python version.") 
```

Comment:
This handling is also superfluous now - because it's all will be guarded by import exception (i.e. if python 2 is used), so again, the `try` blocks has to go.